### DISCUSSION TITLE
access mantine theme in emotion styled object

### DISCUSSION THREAD

**Discussion by ghost** [mantinedev/mantine#1395] [Q&A] [answered]
I am trying to access the theme object in an emotion styled component similar to the following:

`const GroupTitle = styled(Title)(({ theme }) => ({
  color: theme.colors.darkBlue,
  marginBottom: '16px',
  textTransform: 'capitalize',
}));`

In order for it to work however, I need to add the @emotion/react ThemeProvider in my nextjs _app.js.  Is that the right way to go about it, or should the component be able to access theme from the MantineProvider?

**Comment by rtivital** [comment, marked as answer]
There is no way to access theme with emotion styled, you can only use `useMantineTheme` hook and pass it though props.
